### DISCUSSION TITLE
In mac or mobile devices button in not showing properly

### DISCUSSION THREAD

**Discussion by Gauravthakur262** [tailwindlabs/tailwindcss#5692] [Help]
Hi,

I have used tailwind css in one of my project. Button is not working properly. In android devices its working fine, but in mobile and Mac button is not showing properly. please see the attached image and  help me to solve this issue
![Button-Issue](https://user-images.githubusercontent.com/42164317/135814825-6a7b5edc-2800-4cd9-a1ee-e9d77a7e74e1.png)
.

**Comment by simonswiss** [comment]
Hey!

What exactly is not working properly? The text not being aligned vertically?

Can you show us some code, so we can help? I don't think anyone can help you solve this issue based on a screenshot.

A great place to make a demo of the problem is here:

https://play.tailwindcss.com

If you cannot reproduce the problem there, it's likely something in your own project. If you can, share that project then, or find a way to reproduce the issue in a minimal situation (minimal repro) and you'll get help much quicker 👍
